packages/dd-trace/test/appsec/index.spec.js

```diff
@@ -66,10 +70,8 @@ describe('AppSec Index', () => {
       AppSec.enable(config)
       AppSec.enable(config)
 
-      expect(fs.readFileSync).to.have.been.calledWithExactly('./path/rules.json')
-      expect(fs.readFileSync).to.have.been.calledWithExactly(config.appsec.blockedTemplateHtml)
-      expect(fs.readFileSync).to.have.been.calledWithExactly(config.appsec.blockedTemplateJson)
-      expect(RuleManager.applyRules).to.have.been.calledOnceWithExactly({ rules: [{ a: 1 }] }, config.appsec)
+      expect(RuleManager.applyRules).to.have.been.calledOnceWithExactly(RULES, config.appsec)
+      expect(blocking.setTemplates).to.have.been.calledOnceWithExactly(config)
```

Comment:
swap ordering of the two lines, we call setTemplates before applyRules, it's a nitpick but it makes everything more readable as a rule of thumb :+1: 
```suggestion
      expect(blocking.setTemplates).to.have.been.calledOnceWithExactly(config)
      expect(RuleManager.applyRules).to.have.been.calledOnceWithExactly(RULES, config.appsec)
```